pkg/geoip/geolite2_null.go

```diff
@@ -0,0 +1,38 @@
+// Package geoip helps in geolocation operations.
+package geoip
+
+import (
+	"io"
+	"net"
+)
+
+// nullGeoLite2 is a structure what stores a geoIp2Reader to a GeoIp2 database.
+type nullGeoLite2 struct{}
```

Comment:
The interface should be `nullGeoIp` as `GeoLite2` refer to the implementation.

I'd have a `NewNullGeoIp()` or `NewNull()` for it.